src/y-axis.js

```diff
@@ -88,16 +92,7 @@ class YAxis extends PureComponent {
             <View style={[style]}>
                 <View style={{ flexGrow: 1 }} onLayout={(event) => this._onLayout(event)}>
                     {/*invisible text to allow for parent resizing*/}
-                    <Text
-                        style={{
-                            opacity: 0,
-                            fontSize: svg.fontSize,
-                            fontFamily: svg.fontFamily,
-                            fontWeight: svg.fontWeight,
-                        }}
-                    >
-                        {longestValue}
-                    </Text>
+                    <Text style={{ opacity: 0, fontSize: svg.fontSize }}>{longestValue}</Text>
```

Comment:
Why were these changed? I'm pretty sure fontFamily and fontWeight affects the size.